SSZipArchive/SSZipArchive.m

```diff
@@ -1112,3 +1108,37 @@ - (NSString *)_hexString
 }
 
 @end
+
+#pragma mark Private tools for security
+
+@implementation NSString (SSZipArchive)
+
+- (NSString *)_sanitizedPath
```

Comment:
This implementation may be replaced by mz_path_resolve from minizip for performances: https://github.com/nmoinvaz/minizip/blob/dev/mz_os.c#L136